setup.py

```diff
@@ -30,10 +30,11 @@
     readfiles_require +
     security_require + [
         'gocept.testing',
-        'mock>=0.8.0, < 4',
+        'mock>=0.8.0, < 4',  # PY2
```

Comment:
Might it be a good idea to use the python 2 requirement here? This would be documentation with potential effect.
```suggestion
        'mock>=0.8.0, < 4 ; python_version=="2.7"',
```

Reply:
This requires to switch to `unitest.mock` for Python 3.
I'll take this route.